Control coverage/Feature/APIManagement.md

```diff
@@ -54,7 +54,7 @@ Use of HTTPS ensures server/service authentication and protects data in transit
         # Refer https://docs.microsoft.com/en-us/powershell/module/az.apimanagement/set-azapimanagementapi
         ```
 
-### Azure Policy or ARM API used for evaluation 
+### Azure Policies or REST APIs used for evaluation
 
 - ARM API to list APIMs and its related property at Subscription level: /subscriptions/{subscriptionId}/providers/Microsoft.ApiManagement/service?api-version=2019-12-01<br />
```

Comment:
if you replace ARM in heading why not here?

Reply:
Replaced ARM from sentences too.